### PR TITLE
Add the GPG doc to simorgh

### PR DESCRIPTION
**Pull request by DemonusPC**
Resolves #58

**Overall change:**  
Added the GPG signing guide. On Thursday we 26th of September we will begin enforcing GPG signatures on committs. 

**Code changes:**
None

---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

File: docs/GPG-Signing-Guide.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# GPG Singning Guide
```
Comment (amywalkerdev) [inline docs/GPG-Signing-Guide.md]:
```suggestion
# GPG Signing Guide
```

**Review by amywalkerdev — CHANGES_REQUESTED** [review]
Just one spelling mistake :D

**Review by dr3 — APPROVED** [review]

**Review by amywalkerdev — APPROVED** [review]
Looks good now!

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -55,6 +56,7 @@ git pull upstream latest
 ```
 
 ### Your work
+> Make sure you have a valid GPG key set up in GitHub. **All commits must be GPG signed** We do have a very small [guide](docs/GPG-Signing-Guide.md) for setting up GPG signing on MacOs.
```
Comment (FK78) [inline CONTRIBUTING.md]:
```suggestion
> Make sure you have a valid GPG key set up in GitHub. **All commits must be GPG signed** We do have a very small [guide](docs/GPG-Signing-Guide.md) for setting up GPG signing on MacOS.
```

**Review by FK78 — APPROVED** [review]

**Pull request merged**